internal/tfimport/importer/resources.go

```diff
@@ -113,3 +115,20 @@ func loadFields(fields []string, interactive bool, configValues ...ProviderConfi
 
 	return fieldsMap, nil
 }
+
+func fillTemplate(templ string, fieldsMap map[string]string) (string, error) {
```

Comment:
we have a helper func for this in the template lib if you want to use that instead

Reply:
What's the function?

Reply:
The closest is https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/internal/template/template.go#L92, feel free to make adjustments or add new funcs if needed.

Reply:
Done. Moved the tests over as well.